ilan/versions.csv

```diff
@@ -1,2 +1,4 @@
 #version,dir|repo|package package ...,dir|repo|package...,...
-8.5.5.10,/opt/IBM/HTTPServer|http://www.ibm.com/software/repositorymanager/com.ibm.websphere.IHSILAN.v85|com.ibm.websphere.IHSILAN.v85_8.5.5010.20160721_0036,/opt/IBM/WebSphere/Plugins|http://www.ibm.com/software/repositorymanager/com.ibm.websphere.IHSILAN.v85|com.ibm.websphere.PLGILAN.v85_8.5.5010.20160721_0036,/opt/IBM/WebSphere/Toolbox|http://www.ibm.com/software/repositorymanager/com.ibm.websphere.WCTILAN.v85|com.ibm.websphere.WCTILAN.v85_8.5.5010.20160721_0036
+8.5.5.10,/opt/IBM/HTTPServer|http://www.ibm.com/software/repositorymanager/com.ibm.websphere.IHSILAN.v85|com.ibm.websphere.IHSILAN.v85_8.5.5010.20160721_0036,/opt/IBM/WebSphere/Plugins|http://www.ibm.com/software/repositorymanager/com.ibm.websphere.PLGILAN.v85|com.ibm.websphere.PLGILAN.v85_8.5.5010.20160721_0036,/opt/IBM/WebSphere/Toolbox|http://www.ibm.com/software/repositorymanager/com.ibm.websphere.WCTILAN.v85|com.ibm.websphere.WCTILAN.v85_8.5.5010.20160721_0036
+8.5.5.9,/opt/IBM/HTTPServer|http://www.ibm.com/software/repositorymanager/com.ibm.websphere.IHSILAN.v85|com.ibm.websphere.IHSILAN.v85_8.5.5009.20160225_0435,/opt/IBM/WebSphere/Plugins|http://www.ibm.com/software/repositorymanager/com.ibm.websphere.PLGILAN.v85|com.ibm.websphere.PLGILAN.v85_8.5.5009.20160225_0435,/opt/IBM/WebSphere/Toolbox|http://www.ibm.com/software/repositorymanager/com.ibm.websphere.WCTILAN.v85|com.ibm.websphere.WCTILAN.v85_8.5.5009.20160225_0435
+9.0.0.0,/opt/IBM/HTTPServer|http://www.ibm.com/software/repositorymanager/com.ibm.websphere.IHS.v90|com.ibm.java.jdk.v8 com.ibm.websphere.IHS.v90_9.0.0.20160526_1854,/opt/IBM/WebSphere/Plugins|http://www.ibm.com/software/repositorymanager/com.ibm.websphere.PLG.v90|com.ibm.java.jdk.v8 com.ibm.websphere.PLG.v90_9.0.1.20160906_0049,/opt/IBM/WebSphere/Toolbox|http://www.ibm.com/software/repositorymanager/com.ibm.websphere.WCT.v90|com.ibm.java.jdk.v8 com.ibm.websphere.WCT.v90_9.0.0.20160526_1854
```

Comment:
Does this not mean that we now end up with three copies of Java installed in the image?